src/App.jsx

```diff
@@ -1,43 +1,40 @@
 import React, { useEffect } from "react";
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { HashRouter } from 'react-router-dom'
-import { ToastContainer } from 'react-toastify'
-import 'react-toastify/dist/ReactToastify.css'
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { HashRouter } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
-import Header from './components/Header';
-import Footer from './components/Footer';
+import Header from "./components/Header";
+import Footer from "./components/Footer";
 
-import PlanView from './components/plan/PlanView';
-import PlanForm from './components/plan/PlanForm';
-import PlanShowParticipants from './components/plan/PlanShowParticipants';
+import PlanView from "./components/plan/PlanView";
+import PlanForm from "./components/plan/PlanForm";
 
-import { getAuthToken } from './comutils'
+import { getAuthToken } from "./comutils";
 
 function App() {
+    useEffect(() => {
+        getAuthToken().then((token) => {
+            console.log(token);
+            //toast.success('account created')
+        });
+    }, []);
 
-  useEffect(() => {
-    getAuthToken().then((token) => {
-      console.log(token)
-      //toast.success('account created')
-    })
-  },[])
-
-  return (
-    <HashRouter>
-      <div className='bg-malker-100 min-h-screen'>
-        <Header/>
-        <div className='pt-5 min-h-[88vh]'>
-          <ToastContainer toastStyle={{ backgroundColor: '#edf8fd' }}/>
-            <Routes>
-              <Route exact path="/" element={<PlanForm />} />
-              <Route path="/plan-view/:id" element={<PlanView />} />
-              <Route path="/show-participants" element={<PlanShowParticipants />} />
```

Comment:
It was necessary to do this change, but for the next time please make other PR to make these kind of changes.

Reply:
My bad ! I keep it in mind